src/enhancedScatterChartSettingsModel.ts

```diff
@@ -324,6 +346,17 @@ export class EnhancedScatterChartSettingsModel extends Model {
         this.enableLegendCardSettings, this.enableCategoryLabelsCardSettings, this.enableFillPointCardSettings,
         this.enableBackdropCardSettings, this.enableCrosshairCardSettings, this.enableOutlineCardSettings];
 
+    setLocalizedOptions(localizationManager: ILocalizationManager) { 
+        this.setLocalizedDisplayName(positionOptions, localizationManager);
+        this.setLocalizedDisplayName(displayUnitsOptions, localizationManager);
+    }
+
+    public setLocalizedDisplayName(options: IEnumMember[], localizationManager: ILocalizationManager) {
+        options.forEach(option => {
+            option.displayName = localizationManager.getDisplayName(option.displayName.toString())
```

Comment:
calling localizationManager.getDisplayName() for a displayName does not make any sense (since it takes a displayName as an argument); options should be of different type

Reply:
DisplayName at first is initialized with displayNameKey string, so that localizationManager then localize it. This approach is used because currently we don't a have in-built localization support for enumerations.